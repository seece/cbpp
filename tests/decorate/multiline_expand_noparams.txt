#macro TESTMACRO
Log(s:"in this article all the names have been changed");
Log(s:"except Keijo's");
#endmacro

TESTMACRO
TESTMACRO