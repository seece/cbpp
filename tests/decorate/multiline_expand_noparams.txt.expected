// #macro TESTMACRO
// Log(s:"in this article all the names have been changed");
// Log(s:"except Keijo's");
// #endmacro

Log(s:"in this article all the names have been changed");
Log(s:"except Keijo's");
Log(s:"in this article all the names have been changed");
Log(s:"except Keijo's");